sync/filter.format.basic_html.yml

```diff
@@ -16,7 +16,7 @@ filters:
     status: true
     weight: -10
     settings:
-      allowed_html: '<i> <sub> <sup> <span class=""> <del> <math> <a href> <b> <br> <table> <caption> <tbody> <thead> <tfoot> <th> <td> <tr> <img src alt width height> <ol> <ul> <li>'
+      allowed_html: '<i> <sub> <sup> <del> <math> <a href> <b> <br> <table> <caption> <tbody> <thead> <tfoot> <th> <td> <tr> <img src alt width height> <ol> <ul> <li>'
```

Comment:
Dropping support for `<span>` should reduce a lot of noise in our markup. This is out of scope but a minor change that will add a lot of value.